### PR TITLE
[T3A5][T15-A4]Bartłomiej Woźniak

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by bartekwozniak93** [comment]
Ready for review

File: src/seedu/addressbook/ui/Formatter.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        showToUser(MESSAGE_GOODBYE, Formatter.DIVIDER, Formatter.DIVIDER);
+    }
+
+
+    public void showInitFailedMessage() {
+        showToUser(MESSAGE_INIT_FAILED, Formatter.DIVIDER, Formatter.DIVIDER);
+    }
+
+    /** Shows message(s) to the user */
+    public void showToUser(String... message) {
```
Comment (nishantbudhdev) [inline src/seedu/addressbook/ui/Formatter.java]: This function doesn't do formatting. How does this violate SRP?

File: src/seedu/addressbook/ui/Formatter.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import seedu.addressbook.commands.CommandResult;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+public class Formatter {
```
Comment (nishantbudhdev) [inline src/seedu/addressbook/ui/Formatter.java]: Missing class comments.

**Comment by nishantbudhdev** [comment]
@bartekwozniak93 Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**